src/components/Post.js

```diff
@@ -62,7 +62,7 @@ export class Post extends React.Component {
 
   static contextTypes = {currentUser: object}
 
-  static childContextTypes = {post: object}
+  static childContextTypes = {post: object, dispatch: func, currentUser: object}
```

Comment:
is this necessary? i would think those two are already available through childContextTypes in the top-level App component